manifests/0000_70_cluster-network-operator_03_configmap.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: v1
```

Comment:
This should be in `bindata`, not `manifests`.

Reply:
I could place it there, sure. But I did not because I thought it would be clearer if this ran in `openshift-network-operator` namespace as I tried to do something generic and independent of the network plugin used. But you might be right as that would allow us to not copy all binaries but be selective about it

Reply:
even more generically - the CVO should not be responsible for deploying dependent resources. Those are *always* the responsibility of the second-level operator.

Reply:
It can run in `openshift-network-operator` but still be in `bindata` rather than `manifests`